src/Goerli-DssSpell.t.sol

```diff
@@ -511,530 +452,29 @@ contract DssSpellTest is GoerliDssSpellTestBase {
-
-        // repay debt and free our collateral
-        rwaurn_009.wipe(daiToPay);
-        rwaurn_009.free(1 * WAD);
-
-        // check if MCD_PAUSE_PROXY have RWA009 Tokens
-        assertEq(rwagem_009.balanceOf(address(this)), 1 * WAD, "RWA009: gem not sent back to the caller");
-
-        // check if we have 0 collateral and outstanding debt in the VAT
-        (ink, art) = vat.urns("RWA009-A", address(rwaurn_009));
-        assertEq(ink, 0, "RWA009: bad `ink` after free()");
-        assertEq(art, 0, "RWA009: bad `art` after wipe()");
-    }
-
-    function testRWA009_OPERATOR_LOCK_DRAW_CAGE() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-
-        uint256 drawAmount = 25_000_000 * WAD;
-
-        // setting address(this) as operator
-        hevm.store(address(rwaurn_009), keccak256(abi.encode(address(this), uint256(1))), bytes32(uint256(1)));
-        assertEq(rwaurn_009.can(address(this)), 1, "RWA009: address(this) not operator");
-
-        // Check if spell lock 1 * WAD of RWA009
-        assertEq(rwagem_009.balanceOf(address(rwajoin_009)), 1 * WAD, "RWA009: wrong gem balance in join");
-
-        // Check if spell draw 25mm DAI to GENESIS
-        assertEq(dai.balanceOf(address(MCD_PAUSE_PROXY)), drawAmount, "RWA009: wrong dai balance in output conduit");
-
-        (uint256 ink, uint256 art) = vat.urns("RWA009-A", address(rwaurn_009));
-        assertEq(ink, 1 * WAD, "RWA009: wrong ink in urn"); // Whole unit of collateral is locked
-        assertEq(art, drawAmount, "RWA009: wrong art in urn"); // DAI drawn == art as rate should always be 1 RAY
-
-        giveAuth(address(end), address(this));
-        end.cage();
-        end.cage("RWA009-A");
-
-        end.skim("RWA009-A", address(rwaurn_009));
-
-        (ink, art) = vat.urns("RWA009-A", address(rwaurn_009));
-        // 100_000_00 is the value attributed to the price of 1*WAD RWA009
-        uint256 skimmedInk = 25_000_000 * WAD / 100_000_000;
-        assertEq(ink, 1 * WAD - skimmedInk, "RWA009: wrong ink in urn after skim");
-        assertEq(art, 0, "RWA009: wrong art in urn after skim");
-
-        hevm.warp(block.timestamp + end.wait());
-
-        vow.heal(min(vat.dai(address(vow)), sub(sub(vat.sin(address(vow)), vow.Sin()), vow.Ash())));
-
-        // Removing the surplus to allow continuing the execution.
-        hevm.store(
-            address(vat),
-            keccak256(abi.encode(address(vow), uint256(5))),
-            bytes32(uint256(0))
-        );
-
-        end.thaw();
-
-        end.flow("RWA009-A");
-
-        giveTokens(address(dai), 1_000_000 * WAD);
-        dai.approve(address(daiJoin), 1_000_000 * WAD);
-        daiJoin.join(address(this), 1_000_000 * WAD);
-
-        vat.hope(address(end));
-        end.pack(1_000_000 * WAD);
+        assertEq(dai.balanceOf(address(MCD_PAUSE_PROXY)), prevBalance + drawAmount, "RWA009/dai-drawn-was-not-send-to-the-recipient");
```

Comment:
I was very confused with this until I realized the pause proxy is set as the output conduit for that urn on Goerli. Still, I think it would be much clearer if that address was referred to in this context as the output conduit. There is a chainlog entry for it too. And it would make the test more portable to mainnet too.

Reply:
https://github.com/makerdao/spells-goerli/pull/96/commits/3f50866c385c5381ffa13684969710d741091b94